language/stdlib/modules/LibraConfig.move

```diff
@@ -25,16 +29,22 @@ module LibraConfig {
     // Accounts with this privilege can modify config of type TypeName under default_address
     resource struct ModifyConfigCapability<TypeName> {}
 
-    /// Will fail if the account is not association root
-    public fun grant_privileges(account: &signer) {
-        Roles::add_privilege_to_account_association_root_role(account, CreateOnChainConfig{});
+    // Will fail if the account is not association root
+    // DD: refactor
+    // public fun grant_privileges(account: &signer) {
+    //     Roles::add_privilege_to_account_association_root_role(account, CreateOnChainConfig{});
+    // }
+
+    public fun assert_has_on_chain_config_privilege(lr_account: &signer) {
```

Comment:
Not sure if this should be in the module or the `Roles` module. I would have preference for `Roles` since it's in the spreadsheet, but I'm also fine with keeping it here if you would prefer.

Reply:
I put these kinds of functions in the modules that used to define the privileges.  Theory:  If someone wants to change it, that person is more likely to be making other changes in the defining module, not Roles.  Also, a new modules can define it's function there instead of changing Roles, also.

But it's easy to move these functions, and putting them all in Roles, which would have the advantage of making them easier to compare with the spreadsheeet.

Reply:
Actually, everything is in Roles. That was a *duplicate* function, in LibraConfig by mistake (fixed).